Executive 2023-09-27

VoteWizard opened the pull request.

MakerJanSky commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/12) "[Request to Poll] Offboarding Legacy Legal Recourse Assets"
+
+If this executive proposal passes, `tell` will be called on the RWA Liquidation Oracle for this vault type; this places the vault into a “soft liquidation”.
```
Is it possible to provide additional information about the specific actions involved in a "soft liquidation"?

MakerJanSky replied:
This post might assist:

https://forum.makerdao.com/t/vault-liquidation-mechanism-for-centrifuge-trade-finance-assets-a-pre-mip-discussion/3737

VoteWizard replied:
I considered adding more details, but I opted for simplicity to ensure clarity for the voters. I also came across this: https://mips.makerdao.com/mips/details/MIP22#phase-2-soft-liquidation
However, the MIP is obsolete, and I'm uncertain about the current relevance of the information there

MakerJanSky replied:
I strongly believe that we need to provide more information here. What exactly is a "soft liquidation"? It's not reasonable to expect MKR holders to understand what will happen after this parameter is called, especially when even Steakhouse didn't have a clear understanding of it. We should include a sentence or two to clarify what Centrifuge will do once this parameter is called. This is particularly important because we haven't provided any details on this in the poll.

For reference, this is what Patrick put on a previous spell "Call tell on the RWALiquidationOracle for RWA003-A - this will allow Centrifuge to begin the unwinding process for this position."

MakerJanSky commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+
+If this executive proposal passes, a [Proxy Spell](https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976) for Spark Protocol implementing the following changes will be triggered at [0x9FfFbc278119Ad854b58C3d219212849E8B54eF8](https://etherscan.io/address/0x9FfFbc278119Ad854b58C3d219212849E8B54eF8):
+
+- Activate the Gnosis Chain instance of Spark Lend.
```
I suggest expanding this sentence. perhaps something like "If this proxy spell is executed, a  Spark Lend will be deployed and activated on the Gnosis Chain, allowing the pre-authorized lending mechanisms to be available."

VoteWizard replied:
Are you sure it hasn't already been deployed, and this spell simply activates it?

MakerJanSky replied:
Ah, that's a valid point, and this should be sufficient.

VoteWizard replied:
Thank you!

MakerJanSky commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+If this executive proposal passes, a [Proxy Spell](https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976) for Spark Protocol implementing the following changes will be triggered at [0x9FfFbc278119Ad854b58C3d219212849E8B54eF8](https://etherscan.io/address/0x9FfFbc278119Ad854b58C3d219212849E8B54eF8):
+
+- Activate the Gnosis Chain instance of Spark Lend.
+
```
I think this post should also be cited:

https://forum.makerdao.com/t/risk-assessment-and-parameter-recommendations-spark-protocol-on-gnosis-chain/21629

VoteWizard replied:
Thank you, good catch!

MakerJanSky reviewed (changes requested):
Not all of them are obligatory, but I think some of my comments necessitate alterations.

VoteWizard commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+It includes the latest [DAO Resolution](https://amaranth-secondary-stingray-939.mypinata.cloud/ipfs/QmZ94FG8YXK4seyBHBi2FfTfW5URtBqbCb7JZAB1HGkTNF)
+
+This DAO Resolution will instruct the trustees of the Monetalis Clydesdale (RWA007-A) to the following JAT1 (James Asset Trust 1) to JAT2 (James Asset Trust 2) Asset Reallocation:
+- Selling the ETF products within JAT1.
```
Thank you JS, I'll add this forum post link so they can see this information: https://forum.makerdao.com/t/assessment-jat1-jat2-reallocation/21987

VoteWizard commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+This DAO Resolution will instruct the trustees of the Monetalis Clydesdale (RWA007-A) to the following JAT1 (James Asset Trust 1) to JAT2 (James Asset Trust 2) Asset Reallocation:
+- Selling the ETF products within JAT1.
+- Paying any proceeds from this sale after expenses in excess of 500 million USD into the surplus buffer.
+- Deploying the 500 million USD towards the T-bills ladder already installed under JAT2.
```
I'll write out "United States Treasury bills" thanks!

VoteWizard commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+
+### Fortunafi (RWA005-A) vault changes
+
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
```
I believe you are correct. I should have used the poll.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+
+If this executive proposal passes, the Monetalis Clydesdale (RWA007-A) [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) will be updated to [QmWo3UVtEDKVwS5k34uLt1J6u9px3rjHYkTLK2rYQ31E3G](https://jade-ltd-quail-880.mypinata.cloud/ipfs/QmWo3UVtEDKVwS5k34uLt1J6u9px3rjHYkTLK2rYQ31E3G).
+
+It includes the latest [DAO Resolution](https://amaranth-secondary-stingray-939.mypinata.cloud/ipfs/QmZ94FG8YXK4seyBHBi2FfTfW5URtBqbCb7JZAB1HGkTNF)
```
Missing period at end of sentence

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+
+## Proposal Details
+
+### Monetalis Clydesdale (RWA007-A) Vault Doc Update
```
Suggest "RWA Agreement Parameter" Update - per LFW's advice to use human friendly language :)

If you agree, change for HV Bank header as well.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+### HV Bank (RWA009-A) Vault Doc Update
+
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/13) "[RWA009] HVBank/MIP21 Token CES Domain Team Assessment", [Forum post](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/16) "Huntingdon Valley Bank: Transaction Documents on Permaweb"
```
Suggest taking out the 2nd Forum post link, because the first one already points to it.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/13) "[RWA009] HVBank/MIP21 Token CES Domain Team Assessment", [Forum post](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/16) "Huntingdon Valley Bank: Transaction Documents on Permaweb"
+
+If this executive proposal passes, the HV Bank (RWA009-A) [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) will be updated to [QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx](https://jade-ltd-quail-880.mypinata.cloud/ipfs/QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx).
```
```suggestion
The recently approved HV Bank (RWA009-A) [DAO Resolution](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023) requires updating the [RWA Agreement parameter](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement). 

If this executive proposal passes, the HV Bank RWA Agreement parameter will be updated to [QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx](https://jade-ltd-quail-880.mypinata.cloud/ipfs/QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx).

```

Le-Bateleur-2023 replied:
If you agree with my suggested edit above, the below section will need to be revised by you.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+
+This resolution is intended to terminate future purchases into the RWA Master Participation Trust.
+
+### Fortunafi (RWA005-A) vault changes
```
Capitalize all words in the header

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/request-to-poll-offboarding-legacy-legal-recourse-assets/21582/12) "[Request to Poll] Offboarding Legacy Legal Recourse Assets"
+
+If this executive proposal passes, `tell` will be called on the RWA Liquidation Oracle for this vault type; this places the vault into a “soft liquidation”.
```
We _just_ literally got the confirmation from Centrifuge, Dewiz and Steakhouse about recommending `tell` today.

So I recommend you set up the inclusion of this item by: updating the Forum thread with an explanation of why calling `tell` is being recommended and included in this Exec Vote. 

I think this is a good thing to clarify for the community bc there was a lot of confusion over this in the past.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+
+### Spark Protocol Proxy Spell
+
+*Authorization*: [Governance Executive 2023-09-18](https://vote.makerdao.com/polling/QmVcxd7J)
```
This is a poll.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+*Authorization*: [Governance Executive 2023-09-18](https://vote.makerdao.com/polling/QmVcxd7J)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/proposal-for-activation-of-gnosis-chain-instance/22098/) "Proposal for Activation of Gnosis Chain Instance"
+
+If this executive proposal passes, a [Proxy Spell](https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976) for Spark Protocol implementing the following changes will be triggered at [0x9FfFbc278119Ad854b58C3d219212849E8B54eF8](https://etherscan.io/address/0x9FfFbc278119Ad854b58C3d219212849E8B54eF8):
```
```suggestion
If this executive proposal passes, a [Proxy Spell](https://forum.makerdao.com/t/executive-proxies-and-spark-protocol/20976) for Spark Protocol will be triggered at [0x9FfFbc278119Ad854b58C3d219212849E8B54eF8](https://etherscan.io/address/0x9FfFbc278119Ad854b58C3d219212849E8B54eF8) to activate the Gnosis Chain instance of Spark Lend.
```

Le-Bateleur-2023 replied:
If you accept the suggestion above, the below content will need to be revised.

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+- The Monetalis Clydesdale (RWA007-A) vault [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated, to include the latest DAO Resolution as detailed below.
```
The Monetalis Clydesdale [Poll](https://vote.makerdao.com/polling/Qmb45PDU#poll-detail) Outcome section at the bottom says: 

"Outcomes
If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:

_- This set of instructions will be included in an upcoming Executive Vote._"

To me, this shows that the DAO Resolution now needs to included and approved in this Executive Vote. The successful Poll, in itself, does not mean the DAO Resolution is approved.

Le-Bateleur-2023 reviewed (commented):
Here are my comments VW!

VoteWizard commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+---
+title: Template - [Executive Vote] DAO Resolution for Monetalis Clydesdale, DAO Resolution for HV Bank, Fortunafi vault changes, Spark Protocol Proxy Spell - September 27, 2023
```
I agree with this. I'll make sure to fix the Exec contents

VoteWizard commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,88 @@
+### Monetalis Clydesdale (RWA007-A) Vault Doc Update
+
+*Authorization*: [Governance Poll 2023-09-18](https://vote.makerdao.com/polling/Qmb45PDU)
+*Proposal*: [Forum post](https://forum.makerdao.com/t/proposal-to-revise-asset-allocation-of-jat1-and-jat2/21718) "Proposal to revise asset allocation of JAT1 and JAT2"
```
Thanks, I missed this!

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+---
+title: Template - [Executive Vote] DAO Resolution for Monetalis Clydesdale, HV Bank On-chain RWA Agreement Update, Fortunafi Vault Change, Trigger Spark Protocol Proxy Spell - September 27, 2023
+summary: Approve DAO Resolution and update on-chain RWA Agreement for Monetalis Clydesdale (RWA007-A); update HV Bank on-chain (RWA009-A) RWA Agreement; place Fortunafi (RWA005-A) vault into soft liquidation state; trigger Spark Protocol Proxy Spell.
```
```suggestion
summary: Approve DAO Resolution and update on-chain RWA Agreement for Monetalis Clydesdale (RWA007-A); update HV Bank (RWA009-A) on-chain RWA Agreement; place Fortunafi (RWA005-A) vault into soft liquidation state; trigger Spark Protocol Proxy Spell.
```

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+- A Monetalis Clydesdale (RWA007-A) vault Dao Resolution will be approved and the [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated, to include this DAO Resolution as detailed below.
```
```suggestion
- A DAO Resolution will be approved for Monetalis Clydesdale (RWA007-A); and the [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated to include this DAO Resolution.
```

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+
+## Proposal Details
+
+### Monetalis Clydesdale (RWA007-A) Vault RWA Agreement Parameter update
```
```suggestion
### Monetalis Clydesdale (RWA007-A) Vault - Approve DAO Resolution & Update RWA Agreement Parameter
```

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+
+If this executive proposal passes, the HV Bank RWA Agreement parameter will be updated to [QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx](https://jade-ltd-quail-880.mypinata.cloud/ipfs/QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx).
+
+### Fortunafi (RWA005-A) Vault Changes
```
```suggestion
### Fortunafi (RWA005-A) Vault Change
```

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+
+### HV Bank (RWA009-A) Vault RWA Agreement Parameter Update
+
+*Authorization*: [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)\
```
```suggestion
*Authorization*: [Executive Vote 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)\
```

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+
+### Fortunafi (RWA005-A) Vault Changes
+
+*Authorization*: [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/Qmcb1c9x) , [Governance Executive 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)\
```
```suggestion
*Authorization*: [Governance Poll 2023-08-28](https://vote.makerdao.com/polling/Qmcb1c9x) , [Executive Vote 2023-09-13](https://vote.makerdao.com/executive/template-executive-vote-stability-scope-parameter-changes-spark-protocol-d3m-parameter-changes-set-fortunafi-debt-ceiling-to-zero-dai-dao-resolution-for-hv-bank-delegate-compensation-and-other-actions-september-13-2023)\
```

Le-Bateleur-2023 commented on governance/votes/Executive vote - September 27, 2023.md:
```diff
@@ -0,0 +1,86 @@
+
+### Spark Protocol Proxy Spell
+
+*Authorization*: [Governance Executive 2023-09-18](https://vote.makerdao.com/polling/QmVcxd7J)\
```
```suggestion
*Authorization*: [Governance Poll 2023-09-18](https://vote.makerdao.com/polling/QmVcxd7J)\
```

Le-Bateleur-2023 reviewed (commented):
Minor revisions.

VoteWizard commented:
Given our tight timeline, I've gone ahead and merged the PR. Please note that I've taken into account and addressed all concerns raised in the comments by the two reviewers. While I understand the importance of formal approvals, the time constraints made it imperative to expedite the process. I appreciate everyone's understanding and efforts
